scripts/main.py

```diff
@@ -63,11 +63,11 @@ def log_details(details, filepath, exception):
     stats['rrtex'] = 0
     stats['converted'] = 0
     stats['failed'] = 0
-    stats['skipped'] = 0
+    # stats['skipped'] = 0
 
     details = {}
     details['failed'] = []
-    details['skipped'] = []
+    # details['skipped'] = []
```

Comment:
Is this something we want to have disabled?

Reply:
Well, there was no need to use within the later try-except-else clause as the else was always being executed after the try block if no exception, therefore all the "converted" files were being listed in the log report with the wrong description.